spark/v3.3/build.gradle

```diff
@@ -87,6 +87,14 @@ project(":iceberg-spark:iceberg-spark-${sparkMajorVersion}_${scalaVersion}") {
     testImplementation project(path: ':iceberg-api', configuration: 'testArtifacts')
     testImplementation project(path: ':iceberg-core', configuration: 'testArtifacts')
     testImplementation project(path: ':iceberg-data', configuration: 'testArtifacts')
+    testImplementation project(':iceberg-aws')
+    testImplementation 'software.amazon.awssdk:s3'
+    testImplementation 'software.amazon.awssdk:dynamodb'
+    testImplementation 'software.amazon.awssdk:kms'
+    testImplementation 'software.amazon.awssdk:glue'
+    testImplementation project(':iceberg-gcp')
+    testImplementation platform('com.google.cloud:libraries-bom')
+    testImplementation 'com.google.cloud:google-cloud-storage'
```

Comment:
Aren't these already pulled in through core?

Reply:
Apologies I didn't follow.

As per my understanding integration (i.e aws / aliyun) takes a dependency on core and the iceberg-spark module doesn't take dependency of them ... later the iceberg-spark-runtime module takes dependency takes `implementation` dependency only on iceberg-aws / iceberg-aliyun ... Iceberg runtime module only has Smoke test.

I tried without these dependencies these tests failed for me. Am I missing something here ?

Reply:
I think i got what you meant here, gcp module has `implementation` dependency on the above so it would already be available to us .. unlike in aws modules where they are compileOnly ... 

made the changes, thanks !